### PR TITLE
Plant release notes for release 0.2.0

### PR DESCRIPTION
**Pull request by lucafavatella**
The formatting can be improved further but I feel it is now good enough for merging. We can polish formatting further in following pull request if needed.

Based on initial proposal by @sennui , with inclusion of feedback from other developers.

### CODE REVIEW

File: doc/RELEASE-NOTES-0.2.0.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+  {peers, []}
+ ]},
+{aehttp,
+ [
+  {swagger_port_external, 9903}
+ ]}
+].
+```
+
+4. Use configs supplied below to start nodes. Start nodes (in order 3. 2. 1.)
```
Comment (sennui) [inline doc/RELEASE-NOTES-0.2.0.md]: configs now are above I think. Or by configs do we mean the command lines?

Comment (lucafavatella) [reply doc/RELEASE-NOTES-0.2.0.md]: well spotted

Comment (lucafavatella) [reply doc/RELEASE-NOTES-0.2.0.md]: I am rephrasing

Comment (lucafavatella) [reply doc/RELEASE-NOTES-0.2.0.md]: Rephrased and force-pushed.

File: doc/RELEASE-NOTES-0.2.0.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    ```
+
+The relevant lines are those containing strings `aec_peers` or `aec_sync`.
+
+For example, if node 1 does not manage to contact nodes 2 and 3 you will see something like the following in the log of node 1:
+```
+2017-10-06 13:05:54.632 [debug] <0.669.0>@aec_peers:ping_peer:402 ping result ({peer,"http://localhost:9903/",0}): {error,"A problem occured"}
+2017-10-06 13:05:54.632 [debug] <0.668.0>@aec_peers:ping_peer:402 ping result ({peer,"http://localhost:9902/",0}): {error,"A problem occured"}
+```
+In this case, assuming nodes 2 and 3 are already up, stop and start node 1.
```
Comment (uwiger) [inline doc/RELEASE-NOTES-0.2.0.md]: should we mention by example how this is done? I.e.
```
bin/epoch stop
ERL_FLAGS="..." bin/epoch start
```

File: doc/RELEASE-NOTES-0.2.0.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+* further research on consensus architecture
+
+The packages for the this release are available at the following location:
+https://github.com/aeternity/epoch/releases/tag/v0.2.0-good-peers
+
+To fully appreciate the newest developments you need to experience syncing nodes and sharing blocks.
+
+The easiest way to achieve it, is to deploy 3 local nodes and see how they communicate.
+Please find detailed information about custom configuration here:
+https://github.com/aeternity/epoch/blob/4d1baf36207dfeff2d2f1a5256d37285574b7457/doc/CUSTOMIZATION.md
```
Comment (uwiger) [inline doc/RELEASE-NOTES-0.2.0.md]: we can shorten the commit part, e.g.
`https://github.com/aeternity/epoch/blob/4d1baf3/doc/CUSTOMIZATION.md`

**Comment by lucafavatella** [comment]
One reviewer approved current version of PR. Other reviewer approved version of PR before minor formatting changes. Merging.
CI did not finish but this PR cannot affect CI.

File: doc/RELEASE-NOTES-0.2.0.md (hunk 3 above)
Comment (lucafavatella) [reply doc/RELEASE-NOTES-0.2.0.md]: Addressed in https://github.com/aeternity/epoch/pull/266/commits/24a8d225f2cef992ca7bd6ecf3a37dcbaa3ca80a

File: doc/RELEASE-NOTES-0.2.0.md (hunk 2 above)
Comment (lucafavatella) [reply doc/RELEASE-NOTES-0.2.0.md]: Command for starting node 1 is above so reader shall know that. I plan to add note here only on how to stop - as that is at the end of the notes (so reader cannot know that at this point yet).

Comment (lucafavatella) [reply doc/RELEASE-NOTES-0.2.0.md]: Addressed in https://github.com/aeternity/epoch/pull/266/commits/2871eef0b0e42a32642655ba34248dc724c0a1a6?diff=unified&short_path=70c89f3#diff-70c89f3d71a346f33f5fa3312195cf60

**Pull request merged**